.ci.yaml

```diff
@@ -2375,6 +2375,27 @@ targets:
         ["devicelab", "android", "linux", "pixel", "7pro"]
       task_name: static_path_tessellation_perf__timeline_summary
 
+
+  # Uses Impeller.
+  - name: Linux_pixel_7pro dynamic_path_stroke_tessellation_perf__timeline_summary
```

Comment:
IIRC you'll need to mark these bringup: true

Reply:
The tests were kind enough to notify me. ;)